### PR TITLE
write public announcement post

### LINKED ISSUE #101: Publish official announcement for pivot and public roadmap

### PR DESCRIPTION
**Pull request by alexanderniebuhr**
We wrote this during our `Create Content With Us` event. 
Would love to get some reviews from y'all

cc @florian-lefebvre would love for you to vet this
cc @sarah11918 would love for you to check for good english, if time allows

closes https://github.com/astrolicious/dotdev/issues/101

### CODE REVIEW

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## We are growing 🌱 🌿 🌳
+
+Astrolicious is growing daily, we have gathered over 100 members on our [Discord](https://chat.astrolicious.dev) server and we are excited to see the enthusiasm inside our community. People are sharing their ideas, asking questions and helping each other with their own blog posts, tutorials or examples.
+
+Additionally we have continued our efforts to get more community members involved with decision making, so we are proud to announce that [Fryuni (Luiz Ferraz)](https://github.com/fryuni), has joined the team to shape our future.
+
+## Sunsetting Maintenace Efforts 🌇
+
+Despite the momentum, we have struggled with our dream to offer shared maintenance to our projects. We can't commit to provide maintenance in an amount, we think is necessery for projects to succeed and grow.
```
Comment (florian-lefebvre) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]:
```suggestion
Despite the momentum, we have struggled with our dream to offer shared maintenance to our projects. We can't commit to provide enough maintenance we think is necessary for projects to succeed and grow.
```

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+We have made a tough decision. We are sunsetting our shared code maintance efforts with immediate effect. That means there are not any Astrolicious maintained code project anymore. But our current [projects](/projects) will stay part of Astrolicious and will continue to be maintained by their original authors and engaged community members.
+
+Astrolicious doubles down on content creation and community building. We will offer a Hub to connect the ecosystem, bringing together project authors, Astro users, and content creators. We will continue to create content for Astro Tips, offer a _validated_ [projects list](/projects), help content creators to get discovered, and explore new ways to provide valuable content to our own and Astro community.
```
Comment (florian-lefebvre) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]: I would make this a new section with more details

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Building in Public 📢
+
+We have decided to share as much as possible about our journey publicly. This starts with our [public roadmap](https://github.com/orgs/astrolicious/projects/1) and public [organisational meetings](https://discord.com/events/1217527207467946087/1285303597181177958). We are not stopping there, we will continue to share our learnings, insights and decisions transparently on this [blog](/blog). Additionally we are planning to have weekly content creation events on our Discord, to engage with our community.
```
Comment (florian-lefebvre) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]: Worth linking to the events?

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    url: https://github.com/florian-lefebvre
+---
+
+## We are growing 🌱 🌿 🌳
+
+Astrolicious is growing daily, we have gathered over 100 members on our [Discord](https://chat.astrolicious.dev) server and we are excited to see the enthusiasm inside our community. People are sharing their ideas, asking questions and helping each other with their own blog posts, tutorials or examples.
+
+Additionally we have continued our efforts to get more community members involved with decision making, so we are proud to announce that [Fryuni (Luiz Ferraz)](https://github.com/fryuni), has joined the team to shape our future.
+
+## Sunsetting Maintenace Efforts 🌇
```
Comment (ArmandPhilippot) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]: typo:
```suggestion
## Sunsetting Maintenance Efforts 🌇
```

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+Despite the momentum, we have struggled with our dream to offer shared maintenance to our projects. We can't commit to provide maintenance in an amount, we think is necessery for projects to succeed and grow.
+
+We have made a tough decision. We are sunsetting our shared code maintance efforts with immediate effect. That means there are not any Astrolicious maintained code project anymore. But our current [projects](/projects) will stay part of Astrolicious and will continue to be maintained by their original authors and engaged community members.
```
Comment (ArmandPhilippot) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]: typo:
```suggestion
We have made a tough decision. We are sunsetting our shared code maintenance efforts with immediate effect. That means there are not any Astrolicious maintained code project anymore. But our current [projects](/projects) will stay part of Astrolicious and will continue to be maintained by their original authors and engaged community members.
```

File: apps/site/src/content/blog/astrolicious-evolved.mdx (hunk 1 above)
Comment (ArmandPhilippot) [reply apps/site/src/content/blog/astrolicious-evolved.mdx]: Another suggestion (feel free to ignore, I'm not the best person to judge grammar/wording in English 😉 )
```suggestion
Despite the momentum, we have struggled with our dream to offer shared maintenance to our projects. We can't commit to providing the necessary maintenance for projects to succeed and grow.
```

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+authors:
+  - name: Alexander Niebuhr
+    url: https://github.com/alexanderniebuhr
+  - name: Florian Lefebvre
+    url: https://github.com/florian-lefebvre
+---
+
+## We are growing 🌱 🌿 🌳
+
+Astrolicious is growing daily, we have gathered over 100 members on our [Discord](https://chat.astrolicious.dev) server and we are excited to see the enthusiasm inside our community. People are sharing their ideas, asking questions and helping each other with their own blog posts, tutorials or examples.
```
Comment (sarah11918) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]:
```suggestion
Astrolicious is growing daily! We have gathered over 100 members on our [Discord](https://chat.astrolicious.dev) server and we are excited to see the enthusiasm inside our community. People are sharing their ideas, asking questions and helping each other with their own blog posts, tutorials or examples.
```
Either a colon, or start a new sentence like "Astrolicious is growing daily! We have gathered..."

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+Additionally we have continued our efforts to get more community members involved with decision making, so we are proud to announce that [Fryuni (Luiz Ferraz)](https://github.com/fryuni), has joined the team to shape our future.
```
Comment (sarah11918) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]:
```suggestion
Additionally, in our continued our efforts to get more community members involved with decision making, we are proud to announce that [Fryuni (Luiz Ferraz)](https://github.com/fryuni), has joined the team to help shape our future.
```
I love Fryuni as much as the next person, but I *assume* you haven't given him free rein here.... 😅

File: apps/site/src/content/blog/astrolicious-evolved.mdx (hunk 5 above)
Comment (sarah11918) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]:
```suggestion
We have made a tough decision. We are sunsetting our shared code maintenance efforts, effective immediately. This means there are no longer any Astrolicious maintained community code projects. But our own [project initiatives](/projects) will stay part of Astrolicious and will continue to be maintained by their original authors and engaged community members.
```
Not exactly sure how you want to refer to the things that are gone and the things that stay, but that could be one way. I'm assuming you mean "the projects we initiated" are staying and "projects who came to find a home here" are the ones that have left.

File: apps/site/src/content/blog/astrolicious-evolved.mdx (hunk 2 above)
Comment (sarah11918) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]:
```suggestion
## Our new focus

Astrolicious is doubling down on **content creation and community building**.

We will offer a Hub to connect the ecosystem, bringing together project authors, Astro users, and content creators. We will continue to create content for Astro Tips, offer a _validated_ [projects list](/projects), help content creators to get discovered, and explore new ways to provide valuable content to our own and Astro community.
```
That could be the heading Armand was looking for!

File: apps/site/src/content/blog/astrolicious-evolved.mdx (hunk 3 above)
Comment (sarah11918) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]:
```suggestion
We have decided to share as much as possible about our journey publicly. This starts with our [public roadmap](https://github.com/orgs/astrolicious/projects/1) and public [organisational meetings](https://discord.com/events/1217527207467946087/1285303597181177958).

We are not stopping there! We will continue to share our learnings, insights and decisions transparently on this [blog](/blog). Additionally we are planning to have weekly content creation events on our Discord, to engage with our community.
```

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+Astrolicious doubles down on content creation and community building. We will offer a Hub to connect the ecosystem, bringing together project authors, Astro users, and content creators. We will continue to create content for Astro Tips, offer a _validated_ [projects list](/projects), help content creators to get discovered, and explore new ways to provide valuable content to our own and Astro community.
+
+## Building in Public 📢
+
+We have decided to share as much as possible about our journey publicly. This starts with our [public roadmap](https://github.com/orgs/astrolicious/projects/1) and public [organisational meetings](https://discord.com/events/1217527207467946087/1285303597181177958). We are not stopping there, we will continue to share our learnings, insights and decisions transparently on this [blog](/blog). Additionally we are planning to have weekly content creation events on our Discord, to engage with our community.
+
+We are excited to see what the future holds for Astrolicious, hoping y'all join us on this adventure.
```
Comment (sarah11918) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]:
```suggestion
We are excited to see what the future holds for Astrolicious, and are hoping y'all join us on this adventure.
```

**Review by sarah11918 — COMMENTED** [review]
Left you some grammar edits, for your consideration! 💜

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 9]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Astrolicious: Evolved"
```
Comment (florian-lefebvre) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]: wdyt about "Astrolicious evolves"?

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+    url: https://github.com/alexanderniebuhr
+  - name: Florian Lefebvre
+    url: https://github.com/florian-lefebvre
+---
+
+## We are growing 🌱 🌿 🌳
+
+Astrolicious is growing daily! We have gathered over 100 members on our [Discord](https://chat.astrolicious.dev) server and we are excited to see the enthusiasm inside our community. People are sharing their ideas, asking questions and helping each other with their own blog posts, tutorials or examples.
+
+Additionally, in our continued our efforts to get more community members involved with decision making, we are proud to announce that [Fryuni (Luiz Ferraz)](https://github.com/fryuni), has joined the team to help shape our future.
```
Comment (florian-lefebvre) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]:
```suggestion
Astrolicious is growing daily! We have gathered over 100 members on our [Discord](https://chat.astrolicious.dev) server and we are excited to see the enthusiasm inside our community. People are sharing their ideas, asking questions and helping each other with their own blog posts, tutorials or examples.

Additionally, in our continued our efforts to get more community members involved with decision making, we are proud to announce that [Fryuni (Luiz Ferraz)](https://github.com/fryuni) has joined the team to help shape our future.
```

File: apps/site/src/content/blog/astrolicious-evolved.mdx [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+## Sunsetting Maintenance Efforts 🌇
+
+Despite the momentum, we have struggled with our dream to offer shared maintenance to our projects. We can't commit to providing the necessary maintenance for projects to succeed and grow.
+
+We have made a tough decision. We are sunsetting our shared code maintenance efforts, effective immediately. This means there is no longer any Astrolicious code maintenance. But our all the [projects](/projects) will stay part of Astrolicious and will continue to be maintained by their original authors and engaged community members.
+
+## Our new focus
```
Comment (florian-lefebvre) [inline apps/site/src/content/blog/astrolicious-evolved.mdx]: Probably need an emoji there

**Pull request merged**